Makefile

```diff
@@ -59,6 +59,38 @@ E2E_TEST_CONFIG ?=
 
 LDFLAGS = -ldflags "-s -w -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$(VERSION) -X sigs.k8s.io/node-feature-discovery/source.pathPrefix=$(HOSTMOUNT_PREFIX)"
 
+## multi-arch build
+# required to enable buildx
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+# build with buildx
+PLATFORMS ?= linux/amd64,linux/arm64
+PROGRESS   = auto
+# buildx keeps the build in the cache no default output
+# use --push as OUTPUT to push to registry
+# or --load to have it available as an image
+OUTPUT =
+
+# enable buildx
+ensure-buildx:
+	./hack/init-buildx.sh
+
+IMAGE_BUILDX_CMD ?= docker buildx build --platform=${PLATFORMS} $(OUTPUT) --progress=$(PROGRESS)
+
+IMAGE_BUILD_ARGS = --build-arg VERSION=$(VERSION) \
+	    	   --build-arg HOSTMOUNT_PREFIX=$(CONTAINER_HOSTMOUNT_PREFIX) \
+	    	   --build-arg BASE_IMAGE_FULL=$(BASE_IMAGE_FULL) \
+	    	   --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL)
+IMAGE_TAG_FULL = --target full \
```

Comment:
nit: Pondering the name of the var. `IMAGE_BUILD_ARGS_FULL`, WDYT?

Reply:
Yeah, i realized that I needed to move the `--target` thing also into the tag definition so technically it is not a `TAG` only vars.
`IMAGE_BUILD_ARGS_FULL` + `IMAGE_BUILD_ARGS_MINIMAL`?

Reply:
Update @marquiz PTAL